udata/core/reuse/views.py

```diff
@@ -125,6 +134,14 @@ def initialize_form(self, form):
     def on_form_valid(self, form):
         response = super(ReuseCreateView, self).on_form_valid(form)
         notify_new_reuse.delay(self.object)
+        params = {
```

Comment:
Maybe we should encapsulate the action sending because it's always the same f(name, signal).
At least user extraction, remote adresse... will be coded once for all.